tasks/scripting/src/main/java/org/infinispan/scripting/impl/ScriptMetadata.java

```diff
@@ -134,7 +134,7 @@ public String toString() {
       Optional<String> combiner = Optional.empty();
       Optional<String> collator = Optional.empty();
       Optional<String> reducer = Optional.empty();
-      MediaType dataType = MediaType.APPLICATION_OBJECT;
+      MediaType dataType = MediaType.APPLICATION_UNKNOWN;
```

Comment:
I'm not sure if this is the right approach @tristantarrant @gustavonalle, but it does fix `ScriptingDataStoresTest`